src/DynamoCore/DSEngine/EngineController.cs

```diff
@@ -486,6 +494,11 @@ public bool UpdateGraph(ICollection<NodeModel> nodes, out Exception fatalExcepti
 
         internal void ReconcileTraceDataAndNotify()
         {
+            if (liveRunnerServices.Core == null)
+            {
+                throw new ObjectDisposedException("EngineController");
```

Comment:
I'm not convinced that the object being disposed is the only possible way this can have happened, so I'm not sure we should use this sys-exception